docs/usage.rst

```diff
@@ -113,6 +113,44 @@ To enable ``hoverxref`` on a domain, you need to use the config :confval:`hoverx
 indicating which are the domains you desire.
 
 
+Tooltip on glossary terms
+-------------------------
+
+You can add tooltips to glossary terms:
+
+.. code-block:: rst
+
+   See the :term:`sphinx:environment` definition in the glossary.
+
+That will render to:
+
+See the :term:`sphinx:environment` definition in the glossary.
+
+To enable ``hoverxref`` on glossary terms, you need to add ``'term'`` to :confval:`hoverxref_roles`.
+
+
+Tooltip on sphinxcontrib-bibtex cites
+-------------------------------------
+
+If you want to show a tooltip on `sphinxcontrib-bibtex <https://sphinxcontrib-bibtex.readthedocs.io/en/latest/>`_ cites,
+you just need to enable it in :confval:`hoverxref_domains` by adding ``'cite'`` to that list.
```

Comment:
We should standardize where we put this info. It's at the bottom of the previous section, and top here. Probably a subhead like `Enable this feature` or similar?

Reply:
Probably a `Enable this feature` section may be too big for each of them since the content itself is really small. However, standardizing where to describe how to enable them makes sense to me.